pkg/p2p/testbed/kadcast_test.go

```diff
@@ -139,6 +139,9 @@ func assertBroadcastMsgReceived(t *testing.T, cluster []*testNode, sender int, d
 	blob, _ := crypto.RandEntropy(uint32(msgSize))
 	cluster[sender].Broadcast(context.Background(), blob)
 
+	if is_remote {
+		blob = blob[:1000]
```

Comment:
If we need to run a smoke test with payload size of 100bytes (less than 1000) this will panic.